critiquebrainz/db/review.py

```diff
@@ -93,23 +94,12 @@ def get_by_id(review_id):
                    "user".show_gravatar,
                    "user".musicbrainz_id,
                    "user".is_blocked,
-                   created_time.created,
                    license.full_name,
                    license.info_url
               FROM review
               JOIN revision ON revision.review_id = review.id
               JOIN "user" ON "user".id = review.user_id
               JOIN license ON license.id = license_id
```

Comment:
Please make sure that the query is right. I see that the `WHERE` clause is now missing. You can fix the query by adding a `WHERE` clause for the `review_id`. :)